backend/seed.py

```
@@ -26,38 +27,131 @@
+            title=unit_data["title"],
+            order=unit_data["order"],
+            course_id=unit_data["course_id"],
+        )
+        units_to_add.append(unit)
 
+    db.session.bulk_save_objects(units_to_add)
+    db.session.commit()
+    logger.info("Units loaded successfully.")
+
+
+def load_modules():
```

Ruff (0.8.2)

Missing return type annotation for public function `load_modules`

Add return type annotation: `None`

(ANN201)